Integration/ClientIntegration.php

```diff
@@ -310,10 +310,10 @@ public function sendContact(
 
         try {
             $this->validateClient($client, $force);
+            $this->addTrace('contactClient', $this->contactClient->getName());
             $this->addTrace('contactClientId', $this->contactClient->getId());
 
-            $this->validateContact($contact);
-            $this->addTrace('contactClientContactId', $this->contact->getId());
+            $this->validateContact();
```

Comment:
did we want to call this without passing in `$contact` ? 
*EDIT* nevermind- I see that you've attached it to the class (`$this->contact`)...